crates/script/src/simulate.rs

```diff
@@ -112,11 +112,11 @@ impl PreSimulationState {
         // Executes all transactions from the different forks concurrently.
         let futs = transactions
             .into_iter()
-            .map(|transaction| async {
+            .map(|mut transaction| async {
                 let mut runner = runners.get(&transaction.rpc).expect("invalid rpc url").write();
-
                 let zk_metadata = transaction.zk.clone();
-                let tx = transaction.tx();
+                let tx = transaction.tx(); //NOTE(zk): this one is `mut` upstream but doesn't need to be
```

Comment:
regression?

Reply:
Not really, I noticed it a few updates ago, and I think they haven't changed it yet - will submit a PR upstream one of these days